bomb_site:
  site1:
    x: -0.03
    y: -0.36
    z: 0.01
  site2:
    x: 1.48
    y: -1.03
    z: 0.0
patrol_points:
- x: 1.57
  y: -1.34
  z: 0.0
- x: -1.23
  y: -1.18
  z: 0.01
- x: -0.94
  y: 0.83
  z: 0.01
- x: 1.65
  y: 0.95
  z: 0.0
spawn_points:
  CT:
  - x: 1.59
    y: -1.3
    z: 0.0
  - x: -0.09
    y: -0.62
    z: 0.01
  T:
  - x: -1.01
    y: 0.87
    z: 0.01
  - x: -1.81
    y: 1.56
    z: 0.01
